packages/code-du-travail-data/indexing/fetchCdtnAdminDocuments.js

```diff
@@ -112,12 +110,28 @@ export async function getDocumentBySource(source) {
     });
   });
   const docs = await Promise.all(pDocuments);
-  const documents = docs.flatMap((docs) => docs.map(toElastic));
+  const documents = docs.flatMap((docs) =>
+    // /!\ beware /!\
+    // first map should me removed once admin has a way to handle
+    // editorial content's theme / path
+    docs
+      .map((doc) => {
```

Comment:
why not doing that in toElastic directly ?

Reply:
It deserves its own map IMHO